files/brews/mysql.rb

```diff
@@ -1,8 +1,8 @@
 class Mysql < Formula
   desc "Open source relational database management system"
   homepage "https://dev.mysql.com/doc/refman/5.6/en/"
-  url "https://cdn.mysql.com/Downloads/MySQL-5.6/mysql-5.6.29.tar.gz"
-  sha256 "6ac85b75b2dfa8c232725dda25469df37bf4e48b408cc0978d0dfc34c25a817f"
+  url "https://dev.mysql.com/get/Downloads/MySQL-5.6/mysql-5.6.29.tar.gz"
```

Comment:
could you add it as a mirror instead? (like this one: https://github.com/boxen/puppet-mysql/pull/76/files)